content/1.7/back-office/advanced-parameters/Security/customer-sessions-listing.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Listing
+weight: 1
+---
+# SPECS LISTING customer SESSIONS
+
+## Settings wheel
+When clicking on it, 4 options are displayed:
+1) Export
+
+After clicking on it, the list is exported in CSV format.
+The export takes into account filters and sorts that have been done
+
+2) Import
+
+After clicking on it, the user is redirected to the import page in advanced parameters
+
+3) Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+4) Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+## Bulk actions drop-down
+
+This dropdown is enabled only if at least one session is selected, otherwise, it's disabled.
+
+There is 1 option:
+- Delete selection: it deletes the session. The customer can't log back with the session deleted. 
+
+### Select all checkbox
+
+After checking the box, all the products displayed on the page are selected or unselected.
+
+## Listing
+
+### Columns
+
+There are 10 columns:
+
+- ID of the session
+- Customer ID
+- Customer's First name
+- Customer's Last name
+- Customer's Email
+- Last update
+- Actions: Delete the session
```

Comment:
@PierreRambaud Does it delete the session even if it is not outdated ?

Reply:
Yes, it forces the deletion in database and the disconnect